packages/app/src/runner/event-manager.ts

```diff
@@ -400,10 +401,8 @@ export class EventManager {
     this._addListeners()
   }
 
-  isBrowser (browserName) {
-    if (!this.Cypress) return false
-
-    return this.Cypress.isBrowser(browserName)
+  isBrowserFamily (family) {
+    return getRunnerConfigFromWindow()?.browser?.family === family
```

Comment:
This function is expecting a string, but it's being passed an object so this won't ever compare correctly.

Reply:
I'd waffled too many times between an obj and a string parameter - adding type declaration to parameter will help catch this in the future. addressed in https://github.com/cypress-io/cypress/pull/30061/commits/4af0b991cebd18abba4111220e1e6f0c3b3836f5